engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -7156,6 +7192,12 @@ void demon_hunter_t::init_spells()
   {
     active.sigil_of_flame_t31 = get_background_action<sigil_of_flame_t31_t>( "sigil_of_flame_t31" );
   }
+
+  if ( set_bonuses.t31_havoc_2pc->ok() )
+  {
+    throw_glaive_t* throw_glaive_t31 = get_background_action<throw_glaive_t>( "throw_glaive_t31", "", true );
```

Comment:
Need to make sure this action has a cooldown of `0_s`, otherwise we start getting errors.
```suggestion
    throw_glaive_t* throw_glaive_t31 = get_background_action<throw_glaive_t>( "throw_glaive_t31", "", true );
    throw_glaive_t31->internal_cooldown->base_duration = 0_s;
    throw_glaive_t31->cooldown->base_duration              = 0_s;
    throw_glaive_t31->cooldown->charges                        = 0;
```